pkgs/top-level/all-packages.nix

```diff
@@ -23555,6 +23555,8 @@ with pkgs;
 
   openjpeg = callPackage ../development/libraries/openjpeg { };
 
+  openmesh = callPackage ../development/libraries/openmesh { };
```

Comment:
`openmesh` should go under `pkgs/by-name/op/`

Reply:
Thanks for your much quick review. I'm a bit confused since this is my first time adding a new package. Should I move openmesh from development/libraries to pkgs/by-name/op? I placed it there because some development libraries, like openvdb, were also placed there.

Reply:
Yes so the resulting path is `pkgs/by-name/op/openmesh`. It is recommended that new packages go into `by-name` and they are automatically imported so no need to have the line referenced in this review.